### DISCUSSION TITLE
How to create git-log/history graph?

### DISCUSSION THREAD

**Discussion by pvkarthikk** [epezent/implot#648] [Q&A]
I am new to ImGui and ImPlot. I am building vertical graph like below, I am not sure how to use custom rendering to proceed. Is there any example or headsup, Can I get?
![image](https://github.com/user-attachments/assets/c77b86f5-84e8-438b-8ebc-5c8abe8947a3)

**Comment by brenocq** [comment]
@pvkarthikk, do you have any updates on your adventure? 👀

**Comment by pvkarthikk** [comment]
no luck

**Comment by brenocq** [comment]
Yeaaah, it does look a little bit tricky hahaha there's no build in way of doing it, so you would need to draw the lines/circles manually (compute the position of each of them in the plot given the current git tree). Chatting with some LLM may give you some good directions :)
